src/modules/common/store.ts

```diff
@@ -142,7 +146,18 @@ const store = createStore(rootReducer, enhancer) as RootStore
 const builderAPI = new BuilderAPI(BUILDER_SERVER_URL, new Authorization(store))
 const catalystClient = new CatalystClient(PEER_URL, 'Builder')
 
-sagasMiddleware.run(rootSaga, builderAPI, catalystClient)
+const getClientAddress = () => getAddress(store.getState())!
+const getClientAuthAuthority = () => {
+  const auths = getData(store.getState())
+  const address = getAddress(store.getState())
+  return auths[address!]
+}
+
+const builderClientUrl: string = BUILDER_SERVER_URL.replace('/v1', '')
```

Comment:
It was either this or adding a new environment variable.

Reply:
What do you think of adding a comment above this line explaining the situation?